index.js

```diff
@@ -1,114 +1,45 @@
-        if (err) {
-          console.log(chalk.bgRed("Error: Couldn't create variables.env file"));
-          return;
-        }
-      
-        // SUCCESS!!
-        console.log(boxen(getSuccessMessage(projectDir), boxenOptions));
-      });
+log = console.log;
 
-    });
+remakeCliConfig = new Configstore(name, { user: {} });
+remakeServiceHost = 'https://remakeapps.com';
 
-  }
-
-  if (program.updateFramework) {
-    let ncpOptions = {clobber: true, dereference: false, stopOnErr: true};
-    let cwd = process.cwd();
-    let remakeFrameworkPathInApplicationDirectory = path.join(process.cwd(), "_remake");
-    
-    // 1. CHECK IF _remake DIRECTORY EXISTS
-    if (!fs.existsSync(remakeFrameworkPathInApplicationDirectory)) {
-      console.log(chalk.bgRed("Error: Cannot find a _remake directory in this project"));
-      return;
-    }
-
-    // 2. REMOVE OLD _remake DIRECTORY
-    console.log(chalk.bgGreen("(1/2) Removing old _remake directory..."));
-    rimraf(remakeFrameworkPathInApplicationDirectory, function (rimrafError) {
-
-      if (rimrafError) {
-        console.log(chalk.bgRed("Error: Couldn't remove old _remake directory"));
-        return;
-      }
-
-      // 3. GIT CLONE THE ENTIRE FULL STACK STARTER PROJECT INTO THE CURRENT DIRECTORY
-      console.log(chalk.bgGreen("(2/2) Copying framework into _remake directory..."));
-      shell.exec("git clone --depth 1 https://github.com/panphora/remake-framework.git");
-
-      // 4. MOVE THE _remake DIRECTORY TO WHERE THE OLD _remake DIRECTORY WAS
-      let currentRemakeDir = path.join(cwd, "remake-framework/_remake");
-      shell.mv(path.join(cwd, "remake-framework/_remake"), remakeFrameworkPathInApplicationDirectory);
+program
+  .version('v' + version, '-v, --version', 'output the current version')
+  .name('remake')
+  .usage('command [--help]')
 
-      rimraf(path.join(cwd, "remake-framework"), function (rimrafError) {
+program
+  .command('create <projectDir>')
+  .description('Create a new Remake project')
+  .option('-m, --multitenant', 'create a multi tenant Remake app')
+  .action((projectDir, options) => create(projectDir, options));
 
-        if (rimrafError) {
-          console.log(chalk.bgRed("Error cleaning up: Couldn't remove the ./remake-framework directory"));
-          return;
-        }
+program
+  .command('update-framework')
+  .description('Update the Remake framework in your project')
+  .action(() => updateFramework());
 
-        console.log(chalk.bgGreen("Successfully updated Remake!"));
+program
+  .command('deploy')
+  .description('Deploy your Remake app on the Remake server')
+  .action(() => deploy());
 
-      });
+program
+  .command('build')
+  .description('Build your Remake app')
+  .action(() => build());
 
-    });
+program
+  .command('clean')
+  .description('Wipe the local Remake environment including caches and build assets')
```

Comment:
Will this log them out? I'd mention if so

Reply:
No, it's just cleaning the current project.

Reply:
Ok, that's  good